components/homme/src/theta-l/imex_mod.F90

```diff
@@ -130,12 +130,12 @@ subroutine compute_stage_value_dirk(n0,np1,alphadt,qn0,dt2,elem,hvcoord,hybrid,&
 
     ! dirk settings
     maxiter=20
-    deltatol=1e-13  ! exit if newton increment < deltatol
-    !restol=1e-13    ! exit if residual < restol  
+    deltatol=1.0e-13_real_kind  ! exit if newton increment < deltatol
```

Comment:
could you elaborate here? I think code has a lot of consts defined without _real_kind. if precision changes, won't consts be converted properly?

Reply:
No. 1e-13, e.g., will be viewed as single rather than double unless a flag is given.